mbof/models.py

```diff
@@ -89,6 +89,17 @@ def votes(self):
         )
         return voteTotal
 
+    @property
+    def guests(self):
+        guestTotal = reduce(
```

Comment:
Reduce, lambda are great.
The test looks obscure to me, though it is short enough to be readable as is.  Maybe add intension.count method that returns 0 or 1?